modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,635 @@
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_cdc_current[`cdc.current()`] label:beta[]
+|
+| {check-mark}
+| label:deprecated[Deprecated in 5.17] +
+Replaced by: xref:procedures.adoc#procedure_db_cdc_current[`db.cdc.current()`]
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_cdc_earliest[`cdc.earliest()`] label:beta[]
+|
+| {check-mark}
+| label:deprecated[Deprecated in 5.17] +
+Replaced by: xref:procedures.adoc#procedure_db_cdc_earliest[`db.cdc.earliest()`]
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_cdc_query[`cdc.query()`] label:beta[] label:admin-only[]
+|
+| {check-mark}
+| label:deprecated[Deprecated in 5.17] +
+Replaced by: xref:procedures.adoc#procedure_db_cdc_query[`db.cdc.query()`]
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_db_create_setVectorProperty[`db.create.setVectorProperty()`] label:beta[]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.13] +
+Replaced by: xref:procedures.adoc#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty()`]
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_upgrade[`dbms.upgrade()`] label:admin-only[]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.9]
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_upgradestatus[`dbms.upgradeStatus()`] label:admin-only[]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.9]
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_cluster_readreplicatoggle[`dbms.cluster.readReplicaToggle()`] label:admin-only[]
+|
+| {check-mark}
+| label:deprecated[Deprecated in 5.6] +
+Replaced by: xref:procedures.adoc#procedure_dbms_cluster_secondaryreplicationdisable[`dbms.cluster.secondaryReplicationDisable()`].
+|===
+====
+
+=== Configuration settings
+
+.See all deprecated configuration settings
+[%collapsible]
+====
+[options=header, cols="3m,1,1,2"]
+|===
+| Name
+| Community Edition
+| Enterprise Edition
+| Comment
+
+| xref:configuration/configuration-settings.adoc#config_server.db.query_cache_size[`server.db.query_cache_size`]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.7]
+
+| xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.auth_params[`dbms.security.oidc.<provider>.auth_params`]
+|
+| {check-mark}
+| label:dynamic[] label:deprecated[]
+
+| xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.client_id[`dbms.security.oidc.<provider>.client_id`]
+|
+| {check-mark}
+| label:dynamic[] label:deprecated[]
+|===
+====
+
+[role=label--enterprise]
+=== Metrics
+
+.See all deprecated metrics
+[%collapsible]
+====
+[options=header, cols="3m,3"]
+|===
+| Name
+| Comment
+
+| xref:monitoring/metrics/reference.adoc#db-data-metrics[Database data metrics]
```

Comment:
I think we should list all deprecated database data metrics with their replacements where available.

Reply:
Some of them have kind of replacements. Let me find those... For example:

<prefix>.ids_in_use.relationship => "If you want to have a rough metric of how big your graph is, use <prefix>.neo4j.count.relationship instead."
<prefix>.ids_in_use.node => "If you want to have a rough metric of how big your graph is, use <prefix>.neo4j.count.node instead."

Reply:
oh, yeah. Right! This information in in their descriptions. I missed it. Thank you